debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -735,6 +737,7 @@ public OffsetStorageReader offsetStorageReader() {
 
                             // Purposely not marking this method with @Override as it was introduced in Kafka 2.x
                             // and otherwise would break builds based on Kafka 1.x
+                            @Override
```

Comment:
Please read the comment above the newly added line

Reply:
Good catch, my IDE automatically added it.